sgl-kernel/csrc/cpu/CMakeLists.txt

```diff
@@ -45,8 +45,10 @@ set(SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/decode.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/extend.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/gemm.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/gemm_fp8.cpp
```

Comment:
QQ What build commands are we currently using for CMake or setup? Thank you!

Reply:
https://github.com/sgl-project/sglang/pull/6419 is needed to fix the CMake build issue.
The CMake build command to use with https://github.com/sgl-project/sglang/pull/6419 will be:
```
cd sgl-kernel/
cp pyproject_cpu.toml pyproject.toml
pip install -v .
```

Reply:
And can we delete setup_cpu.py after #6419 BTW we don't recommend to use pybind11, because we want to use one whl for all Python versions.

Reply:
@mingfeima I think it's fine that we remove setup_cpu.py after https://github.com/sgl-project/sglang/pull/6419. Do you have any other concern?
@blzheng is checking how to avoid introducing pybind11 and she will update the implementation in https://github.com/sgl-project/sglang/pull/6419. For now, to test the current PR, we need to use setup_cpu.py or apply the patch of https://github.com/sgl-project/sglang/pull/6419 as a workaround.

Reply:
I confirmed offline with Mingfei that we will remove setup_cpu.py after https://github.com/sgl-project/sglang/pull/6419.